site2/docs/under-construction.md

```diff
@@ -0,0 +1,13 @@
+---
+Id: Under-construction
+title: Under construction
+Sidebar_label: 

```

Comment:
please keep this part consistent with metadata in other files
- all in lowercases
- why no value for sidebar_label?
- if you want to add a new chapter, you need to add it to https://github.com/apache/pulsar/blob/master/site2/website/sidebars.json as well

Reply:
There is no sidebar label because this page will only be accessed via page links.